install_dep_packages.sh

```diff
@@ -64,15 +64,12 @@ echo "Number of Parallel threads used: $NUM_THREADS ..."
 echo ""
 
 # Dependencies needed for building netlink library
-if [[ $OS =~ "Fedora" ]]; then
-    sudo dnf install -y pkgconfig
-    sudo dnf install -y libnl3-devel
-elif [[ $OS =~ "Ubuntu" ]]; then
-    sudo apt-get install -y pkg-config
-    sudo apt-get install -y libnl-route-3-dev
+if [ $OS = "Fedora" ]; then
```

Comment:
Can you also run shellcheck on this? This should likely be something as follows:

```
if [ "$OS" == "Fedora" ]; then
```

The single equal sign assigns the value instead of comparing it.

Reply:
== is a bash-specific alias for =, which performs a string (lexical) comparison, but changed it to == to be more clear